examples/training/object_detection/pascal_voc/faster_rcnn.py

```diff
@@ -64,9 +64,31 @@
 eval_ds = tfds.load("voc/2007", split="test", with_info=False)
 
 with strategy.scope():
-    backbone = keras_cv.models.ResNet50(
-        include_top=False, weights="imagenet", include_rescaling=True
-    ).as_backbone()
+    inputs = tf.keras.layers.Input(shape=image_size)
```

Comment:
Let’s make this a helper function so it doesn’t crowd out the more relevant code here. (Same for the other script)

Reply:
Actually the 2 scripts are using different levels of outputs from backbone, so we'd have to create a slightly more generic helper function to return different backbones in this case